hack/non-olm-install/README.md

```diff
@@ -7,19 +7,25 @@
 The `install-gitops-operator.sh` script supports two methods of installation.
 1. Using operator and component images set as environment variables (default method)
 2. Derive the operator and component images from the `ClusterServiceVersion` manifest present in the operator bundle
-**Note**: This method requires podman or docker binary to be available in the PATH environment variable. Use environment variables `USE_BUNDLE_IMG`, `BUNDLE_IMG` and `DOCKER` for this method of installation
+**Note**: Use environment variables `USE_BUNDLE_IMG`, `BUNDLE_IMG` for this method of installation
 
 
 ### Known issues and work arounds
 
-1. Missing RBAC access to update CRs in `argoproj.io` domain 
+1. Missing RBAC access to update CRs in `argoproj.io` domain
 
-Issue: (Now fixed)
+Affected versions:
+- 1.7.4 and older versions
```

Comment:
Added  `1.7.x` and `1.8.x` releases which are actively maintained, so that the fix can be done on a later bug fix versions, like the fix can be made available in `v1.8.4` and `v1.7.5`